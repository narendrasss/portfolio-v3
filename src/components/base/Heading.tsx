import styled from 'styled-components';
import { Text, TextProps } from './Text';

export const Heading = styled(Text).attrs(() => ({
  as: 'h1',
  fontSize: 2,
  fontWeight: 'semibold',
}))<TextProps>``;
