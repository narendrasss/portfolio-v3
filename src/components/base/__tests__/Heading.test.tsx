import React from 'react';
import { render } from 'react-testing-library';
import Heading from '../Heading';

describe('Heading', () => {
  it('renders without crashing', () => {
    const { asFragment } = render(<Heading />);
    expect(asFragment()).toMatchSnapshot();
  });
});
