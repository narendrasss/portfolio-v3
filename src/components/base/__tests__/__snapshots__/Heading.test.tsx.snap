// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Heading renders without crashing 1`] = `
<DocumentFragment>
  <h1
    class="sc-bdVaJa sc-htpNat sc-bxivhb gBkMPw"
    font-size="2"
    font-weight="bold"
  />
</DocumentFragment>
`;
